pom.xml

```diff
@@ -79,7 +79,6 @@
                 <groupId>junit</groupId>
                 <artifactId>junit</artifactId>
                 <version>4.13.1</version>
-                <scope>test</scope>
```

Comment:
![](https://i.pinimg.com/originals/65/24/cf/6524cf78b824146a2bb68bd06edad1ed.jpg)

For the reference, it's because of `testcontainer`'s dependency on `junit`